db/db.js

```diff
@@ -0,0 +1,50 @@
+const mongoose = require('mongoose');
+
+const sampleData = require('./sampleCommentData');
+
+const Schema = mongoose.Schema; 
+
+mongoose.connect('mongodb://localhost/commentsListViewAndSubmission');
+
+const db = mongoose.connection;
+db.on('error', () => console.log('error connecting to mongo'));
+db.once('open', () => console.log('connected to mongo'));
+
+const repliesSchema = new Schema({
+  id: Number,
```

Comment:
suggest that you make id a unique field to prevent duplicates

Reply:
I spent a good amount of time researching how to have the unique id work within the replies array inside the comment but it only works fully when it's not specified to be unique. I'm not sure why. It would indicate there's an 'error saving'.